app/src/main/java/org/mozilla/fenix/collections/CreateCollectionFragment.kt

```diff
@@ -80,6 +88,24 @@ class CreateCollectionFragment : DialogFragment() {
                     CollectionCreationChange.StepChanged(SaveCollectionStep.NameCollection)
                 )
                 is CollectionCreationAction.BackPressed -> handleBackPress(backPressFrom = it.backPressFrom)
+                is CollectionCreationAction.SaveCollectionName -> {
+                    showSavedSnackbar(it.tabs.size)
+                    dismiss()
+                }
+            }
+        }
+    }
+
+    private fun showSavedSnackbar(tabSize: Int) {
+        context?.let { context: Context ->
+            val rootView = context.getRootView()
+            rootView?.let { view: View ->
+                val string =
+                    if (tabSize > 1)
```

Comment:
We should technically use Android resource plurals here for i18n. https://developer.android.com/guide/topics/resources/string-resource#Plurals

Reply:
Cool! I've never used these before, updated!